src/components/pages/download/BrowserTab.astro

```diff
@@ -174,16 +174,7 @@ const Chromiums = [
     .userscript-badge img {
         height: 90%;
     }
-    party
-        extensions
-        unless
-        they
-        are
-        signed
-        (which also requires store approval).
-        As
-        such,
-    it is not possible for us to offer a Firefox extension. .userscript-text {
```

Comment:
how tf did i manage to do this lol

Reply:
shit happens ig. the funny part is that it doesnt even error for me